agent/app/agent_capability_windows.go

```diff
@@ -49,23 +49,24 @@ func (agent *ecsAgent) appendTaskEIACapabilities(capabilities []*ecs.Attribute)
 }
 
 func (agent *ecsAgent) appendFirelensFluentdCapabilities(capabilities []*ecs.Attribute) []*ecs.Attribute {
-	return capabilities
+	return appendNameOnlyAttribute(capabilities, attributePrefix+capabilityFirelensFluentd)
```

Comment:
Instead of adding new code here - can you move these methods to agent_capability.go, so the same is used for unix and windows as well. this will need removal of these methods from agent_capability_unix.go as well.